### PR TITLE
Make script for predicting space usage in a pool

### PR DESCRIPTION
**Pull request by mulkieran**
Resolves https://github.com/stratis-storage/project/issues/270
Supercedes #2507

### CODE REVIEW

**Review by bgurney-rh — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -202,8 +204,10 @@ install-cfg: docs/stratisd.8
 install: $(PROFILETARGET) install-cfg
 	install -Dpm0755 -t $(DESTDIR)$(LIBEXECDIR) target/$(PROFILEDIR)/stratisd
 	install -Dpm0755 -t $(DESTDIR)$(UDEVDIR) target/$(PROFILEDIR)/stratis-utils
-	mv $(DESTDIR)$(UDEVDIR)/stratis-utils $(DESTDIR)$(UDEVDIR)/stratis-str-cmp
-	ln $(DESTDIR)$(UDEVDIR)/stratis-str-cmp $(DESTDIR)$(UDEVDIR)/stratis-base32-decode
+	mv -fv $(DESTDIR)$(UDEVDIR)/stratis-utils $(DESTDIR)$(UDEVDIR)/stratis-str-cmp
+	ln -fv $(DESTDIR)$(UDEVDIR)/stratis-str-cmp $(DESTDIR)$(UDEVDIR)/stratis-base32-decode
+	mkdir -p $(DESTDIR)$(BINDIR)
```
Comment (jbaublitz) [inline Makefile:209]: Can you explain this line? I don't think we should be creating standard directories for users and if they are installing into a non-standard directory, my thought is that it should already be there.

**Review by jbaublitz — CHANGES_REQUESTED** [review]
One initial question. Should we use clap for all executables if we're already including it? I'm going to review this in more detail soon.

**Comment by jbaublitz** [comment]
Fully reviewed. Other than those two comments, this looks good to me!

File: Makefile (hunk 1 above)
Comment (mulkieran) [reply Makefile:209]: You're right about that. I was a little too focused on getting it working in our Jenkins tests, but for real use we'ld better drop it.

**Comment by mulkieran** [comment]
> One initial question. Should we use clap for all executables if we're already including it? I'm going to review this in more detail soon.

I like having one locally defined parser per executable name, it keeps things simpler than, for example, using subcommand. For uniformity, I can set up parsers for the other two executable names. Is that what you were thinking of?

**Comment by mulkieran** [comment]
@jbaublitz If https://github.com/stratis-storage/stratisd/pull/2513/commits/603f39e98fec5896c28bdefca17d9fd84f4caa23 looks good to you, I'll go ahead and do stratis-base32-decode.

**Comment by jbaublitz** [comment]
> @jbaublitz If [603f39e](https://github.com/stratis-storage/stratisd/commit/603f39e98fec5896c28bdefca17d9fd84f4caa23) looks good to you, I'll go ahead and do stratis-base32-decode.

Other than clap arguments being optional by default, this looks good to me! I think we have to explicitly declare that they're required.

**Comment by mulkieran** [comment]
jbaublitz: Ok! I believe this is ready again. Ignore the clippy error, it's bogus.

**Review by jbaublitz — APPROVED** [review]

**Comment by mulkieran** [comment]
Superceded

**Pull request closed without merging**